data/awsrds/v0/component_test.go

```diff
@@ -0,0 +1,364 @@
+package awsrds
```

Comment:
Using mock is good!
I quickly go through it now, and I will check deeper after you finish the first modification.